tests/anp_test_ingress_egress_intersection/anp.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: cluster-allow-example
+spec:
+  priority: 30
+  subject:
+    namespaces: {}
+  ingress:
+    - action: Allow
+      from:
+      - namespaces: {}
+      ports:
+        - portNumber:
+            protocol: UDP
+            port: 5353
+    - action: Deny # deny others
+      from:
+      - namespaces: {}
+  egress:
+    - action: Allow
+      to:
+      - namespaces:
+            matchLabels:
+              kubernetes.io/metadata.name: foo-ns-2

```

Comment:
yes, and it was intended;
 to see that even though the "egress" rule allows all conns to the "foo-ns-2"; only "UDP 3535" is allowed because of the ingress rule

Reply:
ok, maybe this test can be improved with an "egress" for "udp 5000-6000" , and then the intersection from ingress,egress is only "udp 5353".
can also add another port on the ingress rule, so that it's a non trivial intersection.

Reply:
done

Reply:
TCP 2000 is allowed also on egress,
ANP rules are read as is , it is not mentioned in the rule but not restricted by any other egress rule/policy. so it is allowed by default on egress and since allowed also by ingress it is allowed.
(the policy is still in a mode that the "egress" section does not affect the result.)

Reply:
i can add a "deny all" rule to egress; then the intersection will not include TCP 2000

Reply:
done